mod/wikirate/set/all/wikirate.rb

```diff
@@ -19,6 +19,25 @@
     CGI::escape card.name
   end
 
+  def is_number? str
+    true if Float(str) rescue false
+  end
```

Comment:
this is good for now.  Feels like this should be a method on a content object, but we don't have a fantastic framework for that yet.